models/UsersModel.js

```diff
@@ -0,0 +1,16 @@
+var mongoose = require('mongoose');
+
+var modelsRegistro = mongoose.Schema({
+    id: Number,
+    name: String,
+    lastName: String,
+    mail: String,
```

Comment:
please change the name of this property from "mail" to "email"

Reply:
Done!